wolfssl/wolfcrypt/tfm.h

```diff
@@ -282,12 +282,10 @@
 #define FP_NO         0   /* no response */
 
 /* a FP type */
-typedef struct {
+typedef struct fp_int {
     int      used,
              sign;
-#ifdef ALT_ECC_SIZE
     int      size;
```

Comment:
Adding size to the struct for non ALT_ECC_SIZE users seems like a waste.  It also creates a bunch of conditional branches that may effect timing.